lib/worldwide/region.rb

```diff
@@ -441,6 +460,11 @@ def add_zone_to_hash(zone)
       end
     end
 
+    def additional_field_required?(field_name)
+      field = additional_address_fields.find { |f| f["name"] == field_name }
+      field ? field["required"] : false
```

Comment:
If the field yaml does not include a `required:` key, then `field["required"]` will actually return `nil`. Predicate methods should stick to `true/false`. How about we do:

```ruby
field ? !!field["required"] : false
```

The `!!` will convert all truthy values to `true` and falsy values to `false`, including `nil`.

Reply:
nice catch!